test/ELMDependencyHelper.test.ts

```diff
@@ -1,11 +1,16 @@
 import * as ELMDependencyHelper from '../src/ELMDependencyHelper';
```

Comment:
A little more testing on ELMDependencyHelper would help with our "branch" coverage.

Reply:
Looks like the uncovered branch is at line 149 of ELMDependencyHelper

Reply:
Now have full coverage on all new functions in ELMDependencyHelper. Also added JSDoc comments.